.github/workflows/deploy-release-reusable.yaml

```diff
@@ -28,10 +28,6 @@ on:
         description: Identifier used in the distribution artifact and Subversion repository folder filenames (e.g., `logging-parent`)
         required: true
         type: string
-      site-enabled:
-        description: Flag indicating if a website should be generated prior to deployment using `./mvnw site`
-        default: false
-        type: boolean
```

Comment:
@ppkarwasz, note this decision too. If you want to make a release for a project, the project _must_ have a website too. This assumption helps with generating the release notes, which is used in several places; distribution ZIP, email templates, etc.